daisy_workflows/image_import/enterprise_linux/translate.py

```diff
@@ -289,7 +295,8 @@ def DistroSpecific(spec: TranslateSpec):
   # Remove NetworkManager-config-server if it's present. The package configures
   # NetworkManager to *not* use DHCP.
   #  https://access.redhat.com/solutions/894763
-  run(g, ['yum', 'remove', '-y', 'NetworkManager-config-server'])
+  if yum_is_on_path(spec):
+    run(g, ['yum', 'remove', '-y', 'NetworkManager-config-server'])
```

Comment:
Is there a scenario where `NetworkManager-config-server` is installed without yum? Can we check for it's existence without yum and remove it?

Reply:
that is also my question - yum suppsoe to be the standard way to install/remove a package. However, shall we support the non-standard scenario?

Reply:
We can use `rpm --erase NetworkManager-config-server` if `yum` isn't available. I'll update the test image to include this package and re-test.

My original intuition: Since the VM's creator removed `yum`, they intended to disable package modification. So if `NetworkManager-config-server` was present, they intentionally guarded against its removal. 

That said, the person importing the VM to GCP expects the system to be configured with DHCP.

Reply:
"Since the VM's creator removed yum, they intended to disable package modification. So if NetworkManager-config-server was present, they intentionally guarded against its removal.

That said, the person importing the VM to GCP expects the system to be configured with DHCP."

I don't know whether that assumption is correct. DId you discuss with Zach?